### DISCUSSION TITLE
最新版本安装问题

### DISCUSSION THREAD

**Discussion by LYZ10** [chunibyocola/sc-translator-crx#87] [Q&A]
大佬您好，首先感谢您的辛劳付出。我在安装v-4.6.0“显示清单文件丢失或不可读取”，请问如何解决呢？谢谢您

**Comment by chunibyocola** [comment]
请问你使用的浏览器是什么，以及版本是什么。

**Reply by LYZ10** [reply]
用的是edge，最新版本版本 107.0.1418.35 (正式版本) (64 位)
![image](https://user-images.githubusercontent.com/64691532/200502192-f9ba1dae-db02-4f96-86be-b65675c249d5.png)
![image](https://user-images.githubusercontent.com/64691532/200502436-b66da966-d661-4cd5-800b-755c4a7c6111.png)

**Reply by LYZ10** [reply]
麻烦您看看，谢谢

**Reply by chunibyocola** [reply]
这不太可能吧...要不你直接从谷歌商店下载吧，那边已经审核完成了。我不知道你在哪里下载的 v4.6.0 😂， Edge 浏览器我是测试过没问题的，或者你可以直接从 Edge 商店安装，虽然是 v4.5.0。

**Reply by LYZ10** [reply]
哈哈，好咧，我再试试
我是从Github下载的呢

**Reply by LYZ10** [reply]
大佬顺便想问一下，为啥我在用v4.5.0的时候，选中单词没有那个直接“点击翻译的符号”呢？

**Reply by chunibyocola** [reply]
> 我是从Github下载的呢

能问一下怎么下载的吗？我是没有放下载源的，除了从谷歌商店或者 Firefox 商店获取最新版，应该没有其他途径的。

> 为啥我在用v4.5.0的时候，选中单词没有那个直接“点击翻译的符号”呢？

我不太能准确理解“点击翻译的符号”，能更清晰地描述吗，可以顺便截一下“在网页中”的设置的图。

**Comment by LYZ10** [comment]
就是这个小图标
![image](https://user-images.githubusercontent.com/64691532/200505501-c2a2e13e-38eb-4f02-885a-f45c970c177f.png)
😂救命哈哈，我是在这里下载的，对不起（捂脸，小白不懂技术
![image](https://user-images.githubusercontent.com/64691532/200505647-7c805a44-b0c6-408d-9bb2-b9fd2f832355.png)

**Reply by chunibyocola** [reply]
一般是会在里面加上软件包的，不过我偷懒了...没有放。

有几种可能：
- 没有在当前页面开启翻译功能
- 没有在选项里选中按钮（选项页 -> 在网页中 -> 选择文本后展示按钮）
- 在文本预处理中的正则表达式把文本置为空了
- 设置了直接翻译，在这种情况下不会弹出按钮

**Reply by LYZ10** [reply]
好咧，那我得想想办法弄到个v4.6.0了，这电脑不能登chrome~
您提到的那几个可能，我研究了下，不是很看得懂（哭
我是直接安装就用了的，没设置过什么呢
